classes/class-wc-connect-taxjar-integration.php

```diff
@@ -1195,27 +1220,21 @@ private function is_nexus_address_valid( $address ): bool {
 				continue;
 			}
 
-			if ( ! $exists || $value === '' || $value === null ) {
+			if ( ! $exists || empty( $value ) ) {
 				continue;
 			}
 
-			if ( isset( $rules['type'] ) ) {
-				if ( $rules['type'] === 'string' && ! is_string( $value ) ) {
-					$errors[] = "[$field] field must be a string";
-					continue;
-				}
+			if ( 'string' === $rules['type'] && ! is_string( $value ) ) {
+				$errors[] = "[$field] field must be a string";
+				continue;
 			}
 
-			if ( isset( $rules['max_length'] ) && is_string( $value ) ) {
-				if ( strlen( $value ) > $rules['max_length'] ) {
-					$errors[] = "[$field] field exceeds maximum length of {$rules['max_length']}";
-				}
+			if ( is_string( $value ) && strlen( $value ) > $rules['max_length'] ) {
```

Comment:
```suggestion
			if ( isset( $rules['max_length'] ) && is_string( $value ) && strlen( $value ) > $rules['max_length'] ) {
```
To ensure `max_length` array key is set before we use it.